app/models/candidate.rb

```diff
@@ -44,6 +44,10 @@ def public_id
     "C#{id}"
   end
 
+  def load_tester?
+    email_address.include?('@loadtest.example.com')
```

Comment:
worth checking it's a non prod env? Just in case?

Reply:
Yes, given this is being merged into the `load-test` branch, I think I'll replace the feature flag with a prod env check.